### PR TITLE
Translate documents to french

### LINKED ISSUE #4: Translate GSRI Code of Conduct to French

### LINKED ISSUE #5: Translate standard contribution guide to French

### PR DESCRIPTION
**Pull request by ImplicitDeny**
This PR will add french versions of the code of conduct and contribution guidelines.
- Fix #4 
- Fix #5

### CODE REVIEW

File: CODE_OF_CONDUCT_FR.md [hunk 1]
```diff
@@ -4,4 +4,68 @@
...
\ No newline at end of file
+## Qui sommes-nous ?
+
+Le `Groupe Special de Reconnaissance et d'Intervention` est une équipe ArmA 3 française, orientée milsim. Nous sommes une organisation sans but lucratif composée de membres bénévoles, qui contribuent au développement d'outils et de mods pour le jeu Arma 3.
+
+Nous jouons des missions, privées comme publiques, orientées forces spéciales, à un haut niveau de discipline et de réalisme. Davantage d'information est disponible sur notre [site internet officiel](https://www.gsri.team) et sur notre [serveur Discord](https://discord.gg/bhMn4jd). L'un comme l'autre sont consultables uniquement en français.
+
+## Notre engagement
+
+Dans le but de favoriser un environnement ouvert et accueillant, nous, contributeurs, nous engageons à faire de notre participation dans notre projet et notre communauté une expérience libre de tout harcelement pour tout invididu.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:15]:
```suggestion
Dans le but de favoriser un environnement ouvert et accueillant, nous, contributeurs, nous engageons à faire en sorte que la participation de chacun dans nos projets et notre communauté ne soit pas source de harcèlement.
```

File: CODE_OF_CONDUCT_FR.md [hunk 2]
```diff
@@ -4,4 +4,68 @@
...
+
+## Note importante sur le contenu violent
+
+Cette communauté est rassemblée autour d'un jeu vidéo **catégorisé M par l'ESRB**, du fait de la présentation *de violence intense, de sang, de langage grossier*. C'est pourquoi notre communauté fait l'objet d'une modération appropriée pour des personnes agées de 17 ans et plus. Cela ne singifie pas qu'elle est inappropriée pour des personnes plus jeunes, mais du contenu susceptible de heurter leur sensibilité risque d'être partagé. Par ailleurs, le contenu inapproprié ne sera pas modéré aussi rapidement et efficacement que ce que l'on serait en droit d'attendre d'une équipe de modération travaillant pour une communauté entièrement publique.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:19]:
```suggestion
Cette communauté est rassemblée autour d'un jeu vidéo **catégorisé M par l'ESRB**, du fait de la présentation *de violence intense, de sang, de langage grossier*. C'est pourquoi notre communauté fait l'objet d'une modération appropriée pour des personnes agées de 17 ans et plus. Cela ne singifie pas qu'elle est inappropriée pour des personnes plus jeunes, mais du contenu susceptible de heurter leur sensibilité risque d'être partagé. Par ailleurs, le contenu inapproprié ne sera pas modéré aussi rapidement et efficacement que ce que l'on serait en droit d'attendre d'une équipe de modération travaillant pour une communauté tout public.
```

File: CODE_OF_CONDUCT_FR.md [hunk 3]
```diff
@@ -4,4 +4,68 @@
...
+
+Si vous vous souciez du fait que vous ou vos enfants puissiez être exposé à un tel contenu, nous vous encourageons à vous éloignez de notre contenu, ou à faire le nécessaire pour éviter à vos enfants de consulter notre contenu.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:21]:
```suggestion
Si vous vous êtes gênés par la possibilité que vous ou vos enfants puissiez être exposé à un tel contenu, nous vous encourageons à vous en éloigner, ou à faire le nécessaire pour éviter que vos enfants y accèdent.
```

File: CODE_OF_CONDUCT_FR.md [hunk 4]
```diff
@@ -4,4 +4,68 @@
...
+
+## Règles de bonne conduite
+
+### Comportement respectueux
+
+* **Soyez poli.** De simples expressions telles que *Bonjour*, *S'il-vous-plait*, *Merci*, ou encore *Au revoir* contribuent grandement à faire de cet espace un lieu accueillant et inclusif.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:27]:
```suggestion
* **Soyez poli.** De simples expressions telles que *Hello*, *Please*, *Thank you*, ou encore *Good bye* contribuent grandement à faire de cet espace un lieu accueillant et inclusif.
```

File: CODE_OF_CONDUCT_FR.md [hunk 5]
```diff
@@ -4,4 +4,68 @@
...
+
+* **Surveillez vos écrits.** Les correcteurs orthographiques sont de nos jours très communs. Commettre de trop nombreuses fautes vous fera apparaître comme nonchalant et irrespectueux. Utilisez la ponctuation pour faciliter la lecture de vos messages. Evitez l'emploi de textes en majuscules, qui seront interprétés comme si vous hurliez, et considérés grossiers. Espacez vos paragraphes mais n'abusez pas des sauts de ligne.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:29]:
```suggestion
* **Soyez attentif à votre rédaction.** Les correcteurs orthographiques sont de nos jours très communs. Commettre de trop nombreuses fautes vous fera apparaître comme nonchalant et irrespectueux. Utilisez la ponctuation pour faciliter la lecture de vos messages. Évitez l'emploi de textes en majuscules, qui seront interprétés comme si vous hurliez, et considérés comme grossier. Aérez vos contenus mais n'abusez pas des espacements.
```

File: CODE_OF_CONDUCT_FR.md [hunk 6]
```diff
@@ -4,4 +4,68 @@
...
+
+* **Ne faites pas de hors-sujet.** Chaque espace de notre communauté est prévu pour un unique sujet. Si vous souhaitez référencer plusieurs sujets, créez un nouveau fil de discussion, et ayez recours au [Github Autolink](https://help.github.com/en/articles/autolinked-references-and-urls) pour les référencer.
+
+* **Terminez vos conversations.** En débuter une et ne plus répondre est considéré comme grossier, aussi pensez à le signaler lorsque vous n'avez plus l'intention de suivre l'évolution d'une discussion. Cela aidera l'équipe de modération à catégoriser les conversations et les lecteurs à savoir si un auteur attend une réponse ou non.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:33]:
```suggestion
* **Terminez vos conversations.** En débuter une et ne plus répondre est considéré comme grossier, aussi pensez à signaler lorsque vous n'avez plus l'intention de suivre l'évolution d'une discussion. Cela aidera l'équipe de modération à catégoriser les conversations, et les lecteurs à savoir si les conversations peuvent être poursuivies ou non.
```

File: CODE_OF_CONDUCT_FR.md [hunk 7]
```diff
@@ -4,4 +4,68 @@
...
+
+* **Abordez les conflits avec respect.** Le conflit est un phénomène social naturel, qui peut permettre le progrès s'il est traité correctement. Evitez de résoudre des conflits personnels en public. Astreignez-vous à n'argumenter que sur le fond du sujet et n'employez pas d'argument _ad hominem_. Les messages personnels rendus publics dans le but de discréditer une autre personne seront supprimés sans avertissement. Si une personne vous atteint en privé, vous devez [la bloquer](https://help.github.com/en/articles/blocking-a-user-from-your-personal-account). Notre équipe d'administration est à votre disposition si vous souhaitez une assistance afin de vous protéger d'autrui.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:35]:
```suggestion
* **Abordez les conflits avec respect.** Le conflit est un phénomène social naturel, qui peut permettre de s’améliorer s'il est traité correctement. Évitez de résoudre des conflits personnels en public. Ne détournez pas la conversation de l'objet du conflit et n'employez pas d'_ad hominem argumentum_. Les messages personnels rendus publics dans le but de discréditer une autre personne seront supprimés sans avertissement. Si une personne vous persécute en privé, vous devez [la bloquer](https://help.github.com/en/articles/blocking-a-user-from-your-personal-account). Notre équipe d'administration est à votre disposition si vous souhaitez de l'aide pour vous protéger des agressions.
```

File: CODE_OF_CONDUCT_FR.md [hunk 8]
```diff
@@ -4,4 +4,68 @@
...
+### Bonnes pratiques pour la contribution
+
+* **Communiquez en anglais.** Nous sommes une communauté francophone, mais notre objectif est de contribuer à la communauté ArmA dans son ensemble. L'usage de l'anglais permet aux joueurs d'autres pays et parlant différentes langues de réutiliser nos outils et contribuer à leur amélioration.
+
+* **Ne surchargez pas les conversations.** Ne répondez que si vous avez du contenu à proposer. Si vous souhaitez réagir aux messages d'autres contributeurs, vous devez avoir recours aux émojis. Nous utilisons la convention suivante :
+  * :+1: `:+1:` signifie **oui / d'accord**
+  * :-1: `:-1:` signifie **non / en désaccord**
+  * :hand: `:hand:` signifie **je suis volontaire**
+  
+* **Respectez le style de programmation.** Les directives officielles comme informelles permettent de préserver la lisibilité et facilitent la maintenance.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:46]:
```suggestion
* **Respectez le style de programmation.** Les pratiques officielles comme informelles permettent de préserver la lisibilité et facilitent la maintenance.
```

File: CODE_OF_CONDUCT_FR.md [hunk 9]
```diff
@@ -4,4 +4,68 @@
...
+
+* **Commentez votre code.** Soyez toutefois raisonnable sur la quantité de commentaires, souvenez-vous que le code concerne les *comment ?* tandis que les commentaires répondent aux *pourquoi ?*. Si votre code ne contient aucun commentaire, ou à l'inverse en contient beaucoup, vous faites probablement quelque chose de travers.
+
+* **Agissez en cohérence avec la communauté.** Cela permet d'éviter les gaspillages d'énergie sur un sujet déjà en cours de traitement, ainsi que de prioriser les chantiers les plus critiques. Le meilleur espace pour s'organiser est le [serveur Discord officiel du GSRI](https://discord.gg/bhMn4jd).
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:50]:
```suggestion
* **Coordonnez-vous avec la communauté.** Cela permet d'éviter les gaspillages d'énergie sur un sujet déjà en cours de traitement, ainsi que de prioriser les chantiers les plus critiques. Le meilleur espace pour s'organiser est le [serveur Discord officiel du GSRI](https://discord.gg/bhMn4jd).
```

File: CODE_OF_CONDUCT_FR.md [hunk 10]
```diff
@@ -4,4 +4,68 @@
...
+
+* **Assurez-vous toujours de la compatibilité des licences.** Nos projets sont protégés par différentes licences, en fonction de la nature et de l'objectif du produit. Assurez-vous d'avoir pris connaissance de la licence utilisée avant de contribuer. Si vous souhaitez inclure du contenu tier dans vos contributions, vérifiez que la licence du projet est compatible avec votre contenu. Dans le doute, n'hésitez pas à solliciter un avis.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:52]:
```suggestion
* **Assurez-vous toujours de la compatibilité des licences.** Nos projets sont protégés par différentes licences, en fonction de la nature et de l'objectif du contenu. Assurez-vous d'avoir pris connaissance et d'avoir accepté les termes de la licence utilisée avant de contribuer. Si vous souhaitez inclure du contenu tiers dans vos contributions, vérifiez que la licence du projet est compatible avec celle qui le protège. Dans le doute, n'hésitez pas à solliciter un avis.
```

File: CODE_OF_CONDUCT_FR.md [hunk 11]
```diff
@@ -4,4 +4,68 @@
...
+
+## Portée
+
+Ce code de bonne conduite s'applique à tous les projets hébergés sur Github par l'organisation `gsri-team`.
+
+## Application
+
+La violation du présent code de bonne conduite donnera lieu à une réaction jugée appropriée par l'équipe de modération, et notament :
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:60]:
```suggestion
La violation du présent code de bonne conduite donnera lieu à une réaction jugée appropriée par l'équipe de modération, et notamment :
```

File: CODE_OF_CONDUCT_FR.md [hunk 12]
```diff
@@ -4,4 +4,68 @@
...
+* Rappel à l'ordre
+* Suppression de contenu
+* Blocage temporaire
+* Bannissement permanent
+ 
+Nous ne rendons pas compte des actions de modération. Les rappels à l'ordre seront fait publiquement, nous n'aurons aucune conversation privée sur le sujet. Tout contenu questionnant la modération sera supprimé sans avertissement. Les remarques qualifiant la modération de trop zélée ou à l'inverse laxiste, conduiront à un bannissement immédiat et permanent.
+
+## Signaler une violation
+
+Le meilleur espace pour prendre contact avec nous est [notre serveur Discord](https://discord.gg/bhMn4jd). Vous pouvez cliquer sur ce lien ou visister [notre site web www.gsri.team](https://www.gsri.team) où un lien permanent vers Discord est affiché au dessus de toutes nos pages. Les violations peuvent être signalées dans n'importe quel canal écrit, bien que nous vous encouragions à privilégier le canal `#discussions`. Merci de mentionner `@Cadre` (rassemblant les responsables de l'équipe) de manière à nous transmettre l'information aussi vite que possible. Les canaux vocaux sont également disponibles.
```
Comment (ArwynFr) [inline CODE_OF_CONDUCT_FR.md:71]:
```suggestion
Le meilleur espace pour prendre contact avec nous est [notre serveur Discord](https://discord.gg/bhMn4jd). Vous pouvez cliquer sur ce lien ou visister [notre site web www.gsri.team](https://www.gsri.team) où un lien permanent vers Discord est affiché au dessus de toutes nos pages. Les violations peuvent être signalées dans n'importe quel canal écrit, bien que nous vous encouragions à privilégier le canal `#discussions`. Merci de mentionner `@Cadre` de manière à nous transmettre l'information aussi vite que possible. Les canaux vocaux sont également disponibles.
```

File: CONTRIBUTING_FR.md [hunk 13]
```diff
@@ -4,4 +4,137 @@
 
 ***Note :** ce document est une traduction du [GSRI Standard Contribution Guide](./CONTRIBUTING.md). En cas d'incohérence entre les deux documents, c'est la version anglophone qui s'applique. Cette version est fournie pour faciliter la compréhension mais n'a pas la valeur probante du document anglophone.*
 
-👷 ***Nous sommes actuellement en train de traduire ce document***
\ No newline at end of file
+Cette documentation offre des lignes directrices et des procédures standards pour contribuer à nos projets. Il s'agit d'une documentation générique susceptible d'être remplacée au niveau de chaque projet. Merci de vérifier le fichier `.guithub\CONTRIBUTING.md` du projet concerné pour davantage d'instructions.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:7]:
```suggestion
Cette documentation offre des lignes directrices et des procédures standard pour contribuer à nos projets. Il s'agit d'une documentation générique susceptible d'être remplacée au niveau de chaque projet. Merci de vérifier le fichier `.github\CONTRIBUTING.md` du projet concerné pour davantage d'instructions.
```

File: CONTRIBUTING_FR.md [hunk 14]
```diff
@@ -4,4 +4,137 @@
...
+**Note :** afin de contribuer au code source vous devez posséder un compte _github.com_.
+
+## Comment transmettre des idées et retours d'expérience
+
+La meilleure manière de procéder est d'ouvrir une `issue` sur le dépôt Github concerné.
+Pour cela, rendez-vous sur le dépôt en question, et cliquez sur l'onglet `issues` en dessous du nom du dépôt.
+Cliquez ensuite sur le bouton vert indiquant `New issue`, qui vous donnera accès au formulaire de création.
+
+Avant d'envoyer votre `issue`, vérifiez que vous avez indiqué les éléments suivants :
+* un titre significatif
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:25]:
```suggestion
* un titre éloquent
```

File: CONTRIBUTING_FR.md [hunk 15]
```diff
@@ -4,4 +4,137 @@
...
+* une description détaillée
+* un label `bug` ou encore `enhancement`
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:27]:
```suggestion
* un label `bug` (incident) ou `enhancement` (amélioration)
```

File: CONTRIBUTING_FR.md [hunk 16]
```diff
@@ -4,4 +4,137 @@
...
+
+Si vous avez besoin d'aide dans ce processus, n'hésitez pas à venir sur notre serveur Discord.
+
+## Contribution ponctuelle au code source
+
+**Note :** *cette procédure s'adresse aux contributeurs ponctuels hors-GSRI. Vous n'êtes pas autorisé à écrire directement dans le dépôt, mais vous pouvez soumettre une `pull request` depuis un `fork` de notre dépôt.*
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:33]:
```suggestion
**Note :** *cette procédure s'adresse aux contributeurs ponctuels hors-GSRI. En tant que tel, vous n'êtes pas autorisé à écrire directement dans le dépôt, mais vous pouvez soumettre une `pull request` depuis un `fork` de notre dépôt.*
```

File: CONTRIBUTING_FR.md [hunk 17]
```diff
@@ -4,4 +4,137 @@
...
+
+### Cloner le dépôt
+
+Vous devez utiliser un client git afin de télécharger et synchroniser votre travail avec le dépôt. Si vous n'êtes pas coutumier de git, nous vous recommandons l'usage de [Github Desktop](https://desktop.github.com/).
+
+Une fois votre client git installé, rendez-vous sur la page de votre `fork` sur github.com, et cliquez sur le bouton vert `Clone or download` sur la droite de la page, puis cliquez sur `Open in Desktop` dans l'encart. Github Desktop démarrera alors et vous demandera où télécharger le code source. Sélectionner le dossier de votre choix puis cliquez sur `Clone` ; Github Desktop téléchargera les sources sur votre ordinateur.
+
+### Modifier les sources
+
+Vous pouvez alors travailler sur les sources à l'aide de votre éditeur favoris. Si vous n'en avez pas encore, nous vous conseillons [Microsoft Visual Studio Code](https://code.visualstudio.com/), qui fonctionne bien avec Github Dekstop. Si vous modifiez une mission, merci de suivre les lignes directrices *Comment contribuer aux missions Arma 3*.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:49]:
```suggestion
Vous pouvez alors travailler sur les sources à l'aide de votre éditeur favori. Si vous n'en avez pas encore, nous vous conseillons [Microsoft Visual Studio Code](https://code.visualstudio.com/), qui fonctionne bien avec Github Dekstop. Si vous modifiez une mission, merci de suivre les indications fournis dans le chapitre *Comment contribuer aux missions Arma 3*.
```

File: CONTRIBUTING_FR.md [hunk 18]
```diff
@@ -4,4 +4,137 @@
...
+
+Git dispose de trois différentes fonctions de suivi des changements :
+
+* `git stage` signifie que vous considérez la version actuelle d'un fichier *prêt pour le commit*. C'est une façon efficace d'ajouter ou retirer des changements au commit en cours.
+
+* `git commit` signifie que vos modifications en mode `stage` sont stockées **localement**. Chaque commit est associé à votre identité et vous pouvez ajouter une petite description résumant ce que vous avez modifié et pourquoi. Si vous n'avez rien mis en `stage`, git placera en mode `stage` tous les changements et créera le commit en fonction.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:55]:
```suggestion
* `git commit` signifie que vos modifications en mode `stage` sont stockées **localement**. Chaque commit est associé à votre identité et vous pouvez ajouter une petite description résumant ce que vous avez modifié et pourquoi. Si vous n'avez rien mis en `stage`, git placera en mode `stage` tous les changements détectés et créera le commit en les incluant tous.
```

File: CONTRIBUTING_FR.md [hunk 19]
```diff
@@ -4,4 +4,137 @@
...
+
+### Conserver les dépôts synchronisés
+
+Assurez-vous fréquemment que vous ne travaillez pas sur une version obsolète des sources. Vérifiez que votre branche n'a aucun commit *behind* (en retard) relativement à notre branche `master`. Si votre branche n'est pas à jour avec `upstream/master`, l'ouverture d'une `pull request` sera refusée par notre équipe.
+
+Si vous avez besoin de mettre à jour, vous devriez synchroniser les changements de notre dépôt vers le votre. Cette opération se nomme *intégration*. Durant l'intégration, des conflits peuvent apparaître, par exemple si vous avez modifié une entité qui a été par ailleurs supprimée de notre dépôt par un autre développeur.
+
+La résolution des conflits doit être effectuée sur votre branche de travail, car cela facilite les tests visant à s'assurer que le processus d'intégration n'a pas endommagé vos fonctionnalités. Une fois satisfait de la fusion, vérifiez que votre version du code source fonctionne toujours avant d'ouvrir une `pull request`. Cette technique est nommée *ping-pong merging*.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:65]:
```suggestion
La résolution des conflits doit être effectuée sur votre branche de travail, car cela facilite les tests visant à s'assurer que le processus d'intégration n'a rien cassé. Une fois satisfait de la fusion, vérifiez que votre version du code source fonctionne toujours avant d'ouvrir une `pull request`. Cette technique est nommée *ping-pong merging*.
```

File: CONTRIBUTING_FR.md [hunk 20]
```diff
@@ -4,4 +4,137 @@
...
+
+ **Note : toutes les PR vers d'autres branches que `master` seront refusées.**
+
+Les administrateurs du projet effectueront une revue de votre code avant de fusionner votre travail. Selon le type de projet, des tests et autres opérations automatisées pourront être mises en place afin de s'assurer de la robustesse de votre production. Si un problème apparait, ou si nous remarquons un détail à modifier, vous pourrez ajouter des commits sur votre propre branche de travail afin d'effectuer les corrections nécessaires : elles seront automatiquement ajoutées à votre `pull request`.
+
+## Contribution régulière au code source
+
+**Note :** cette procédure s'adresse aux membres du GSRI ou aux contributeurs réguliers identifiés. Ils·elles sont autorisés à contribuer directement dans notre dépôt afin d'éviter la prolifération des forks. Si vous n'êtes pas autorisés à contribuer directement au dépôt, référez-vous aux instructions de la section *Contribution ponctuelle au code source* plus haut dans ce document.
+
+### Récupérer le droit d'écriture sur le dépôt
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:81]:
```suggestion
### Obtenir le droit d'écriture sur le dépôt
```

File: CONTRIBUTING_FR.md [hunk 21]
```diff
@@ -4,4 +4,137 @@
...
+
+Votre première étape pour les contributions directes est d'obtenir les droits d'accès sur notre dépôt. Prenez contact avec nous sur Discord et nous étudierons votre demande.
+
+### Cloner le dépôt
+
+Si vous êtes un contributeur régulier, vous pouvez contribuer directement au dépôt. Vous n'avez pas besoin de créer de fork de notre dépôt : vous pouvez cloner directement le dépôt qui vous intéresse.
+
+### Créer une nouvelle branche
+
+**Note : cette étape est obligatoire.** Ne pas s'y circonvenir vous permettra certes de créer des commits sur master **localement**, mais toute tentative de `push` vers Github sera automatiquement refusée.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:91]:
```suggestion
**Note : cette étape est obligatoire.** Y circonvenir vous permettra de créer des commits sur master **localement**, mais toute tentative de `push` vers Github sera automatiquement refusée.
```

File: CONTRIBUTING_FR.md [hunk 22]
```diff
@@ -4,4 +4,137 @@
...
+
+Afin de créer une branche, sélectionnez `Branch` / `New branch ...` dans le menu. Github Desktop demandera un nom de branche. Entrez le nom souhaité : pseudo, fonctionnalité... Dans le doute, demandez-nous si le nom choisi est pertinent. Assurez-vous que la branche source est `master`. Une fois créée, publiez la branche sur Github en cliquant sur le bouton dans la barre en haut de Github Desktop.
+
+Les autres opérations sont similaires à celles décrites dans la procédure ci-dessus.
+
+### Garder votre branche à jour avec master
+
+Il est recommandé de vérifier régulièrement que vous ne travaillez pas sur une version obsolète des sources. Vérifiez que votre branche n'a aucun commit *behind* (en retard) relativement à notre branche `master`. Si votre branche n'est pas à jour avec `master`, l'ouverture d'une `pull request` sera refusée par notre équipe. Si des changements ont eu lieu sur la branche `master`, vous pouvez utiliser la fonction `Branch` / `Update from master` de Github Desktop afin de rattraper le retard. La résolution des conflits doit être faite sur votre actuelle branche de travail.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:99]:
```suggestion
Il est recommandé de vérifier régulièrement que vous ne travaillez pas sur une version obsolète des sources. Vérifiez que votre branche n'a aucun commit *behind* (en retard) relativement à notre branche `master`. Si votre branche n'est pas à jour avec `master`, l'ouverture d'une `pull request` sera refusée par notre équipe. Si des changements ont eu lieu sur la branche `master`, vous pouvez utiliser la fonction `Branch` / `Update from master` de Github Desktop afin de rattraper le retard. La résolution des conflits doit être faite sur votre branche de travail.
```

File: CONTRIBUTING_FR.md [hunk 23]
```diff
@@ -4,4 +4,137 @@
...
+
+### Ouvrir une pull request vers master
+
+Cette procédure est également la même que précédemment. Votre dépôt doit être à jour avec `master`, sans quoi votre `pull request` sera refusée. Des tests et autres opérations automatisées pourront également être menées pour s'assurer de la robustesse de votre production. Les fusions sont ensuites faites par le personnel du GSRI. Vous pouvez ajouter des commits à votre branche de travail : ils seront automatiquement ajoutés à la PR en cours.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:103]:
```suggestion
Cette procédure est également la même que pour les contributeurs occasionnels. Votre dépôt doit être à jour avec `master`, sans quoi votre `pull request` sera refusée. Des tests et autres opérations automatisées pourront également être menées pour s'assurer de la robustesse de votre production. Les fusions sont ensuite faites par l'équipe de maintenance du GSRI. Vous pouvez ajouter des commit à votre branche de travail : ils seront automatiquement ajoutés à la PR en cours.
```

File: CONTRIBUTING_FR.md [hunk 24]
```diff
@@ -4,4 +4,137 @@
...
+
+## Information spécifique à propos des contributions aux missions ArmA 3
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:105]:
```suggestion
## Informations spécifiques à propos des contributions aux missions ArmA 3
```

File: CONTRIBUTING_FR.md [hunk 25]
```diff
@@ -4,4 +4,137 @@
...
+
+### Binarisation du fichier de mission
+
+**Vérifier que les fichiers ne sont *pas* binarisés.**
+
+Cela concerne principalement les fichiers de type `mission.sqm`, mais également de type `config.cpp`. Git sait versionner efficacement les fichiers texte, pas les binaires. Cela permet également de garder un meilleur historique des changements. La binarisation du fichier final est prévue par nos outils avant publication sous forme de fichier PBO.
+
+Par ailleurs, n'ajoutez pas de contenu binarisé ou archivé sous forme de PBO par vos propres soins. Nos outils se chargeront d'assembler, de binariser et d'archiver correctement tout le contenu depuis les sources.
+
+Si vos modifications incluent de gros fichiers binaires (images, audio, video), merci de nous contacter afin de déterminer s'il doit être versionné dans Github ou bien être hébergée dans notre cloud privé pour intégration.
```
Comment (ArwynFr) [inline CONTRIBUTING_FR.md:131]:
```suggestion
Si vos modifications incluent de gros fichiers binaires (images, audio, video), merci de nous contacter afin de déterminer s'ils doivent être versionnés dans Github ou bien être hébergée dans notre cloud privé pour intégration.
```

File: CODE_OF_CONDUCT_FR.md (hunk 2 above)
Comment (ImplicitDeny) [reply CODE_OF_CONDUCT_FR.md:19]: If I'm right, there is no change in this suggestion ?

File: CONTRIBUTING_FR.md (hunk 22 above)
Comment (ImplicitDeny) [reply CONTRIBUTING_FR.md:99]: If I'm right, there is no change in this suggestion ?

**Pull request merged**